doc/source/how-to/continuous-integration.rst

```diff
@@ -176,3 +176,354 @@ Workflow examples are provided for checking :ref:`Coding style`,
+| ``PYANSYS_CI_BOT_TOKEN``            | All repositories                   | :ref:`Yes (GitHub) <github_tokens>`   | Token ``PYANSYS_CI_BOT_TOKEN``            | Bot token for general purpose. It has repository read/write permissions and package read.    |
++-------------------------------------+------------------------------------+---------------------------------------+-------------------------------------------+----------------------------------------------------------------------------------------------+
+| ``PYANSYS_PYPI_PRIVATE_PAT``        | All repositories                   | :ref:`Yes (PyPI) <pypi_tokens>`       | Token ``PYANSYS_PYPI_PRIVATE_PAT``        | Token to publish to Ansys private PyPI channel.                                              |
++-------------------------------------+------------------------------------+---------------------------------------+-------------------------------------------+----------------------------------------------------------------------------------------------+
+| ``PYPI_TOKEN``                      | Private and internal repositories  | :ref:`Yes (PyPI) <pypi_tokens>`       | **Empty**                                 | This should be overwritten at each repository after first public release.                    |
++-------------------------------------+------------------------------------+---------------------------------------+-------------------------------------------+----------------------------------------------------------------------------------------------+
+| ``PYPI_TESTING_TOKEN``              | Private and internal repositories  | :ref:`Yes (PyPI) <pypi_tokens>`       | *Secret*                                  | Token for testing publication to PyPI.                                                       |
++-------------------------------------+------------------------------------+---------------------------------------+-------------------------------------------+----------------------------------------------------------------------------------------------+
+| ``TWINE_TOKEN``                     | Private and internal repositories  | :ref:`Yes (PyPI) <pypi_tokens>`       | **Empty**                                 | This should be overwritten at each repository after first public release.                    |
++-------------------------------------+------------------------------------+---------------------------------------+-------------------------------------------+----------------------------------------------------------------------------------------------+
+
+The values of the ``secrets`` can be obtained by emailing pyansys.core@ansys.com.
+
+Dependabot secrets
+~~~~~~~~~~~~~~~~~~
+
+These secrets in most of the cases a replica of the `Actions secrets`_. 
```

Comment:
```suggestion
Dependabout secrets are generally replicas of the `Actions secrets`_. 
```
I didn't realize earlier that you had set up a link directive. I still think using "GitHub Actions secrets" would be better.

Reply:
I updated your comment.